php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -272,7 +272,16 @@ class NDB_BVL_Instrument extends NDB_Page
         $instrumentPermissions = $config->getSetting("instrumentPermissions");
 
         //check if instrumentPermissions are being used.
-        if ($instrumentPermissions['useInstrumentPermissions'] == false) {
+        if (!is_array($instrumentPermissions)) {
```

Comment:
not used since otherwise we get error 
"Config setting instrumentPermissions does not exist in database"